### PR TITLE
1.16.221.1 Support idk if this is correct or not but lmao

### PR DESCRIPTION
**Pull request by LeonetKenner**

### CODE REVIEW

File: Memory/GameData.h [hunk 1]
```diff
@@ -32,7 +32,8 @@ enum GAMEVERSION : int {
 	g_1_16_0 = 0,
 	g_1_16_1 = 1,
 	g_1_16_10 = 10,
-	g_1_16_20 = 20
+	g_1_16_20 = 20,
+	g_1_16_221 = 221
```
Comment (LeonetKenner) [inline Memory/GameData.h:36]: idk iff this will work but ill try to build it with vs code and test it out

**Comment by richardletshacks** [comment]
no

**Comment by LeonetKenner** [comment]
nvm

**Comment by LeonetKenner** [comment]
@richardletshacks  it works!!! can you merge it !!

**Comment by LeonetKenner** [comment]
Screenshot 2021-06-08 175842

**Comment by LeonetKenner** [comment]
the debug build is working but the release is crashing hmmmmm

**Comment by LeonetKenner** [comment]
welp wait debug build not working on servers

**Comment by LeonetKenner** [comment]
i guess itll be on 1.16.201

**Comment by LeonetKenner** [comment]
nope not even close my code only works on 1.16.221 and versions below 1.16.210.05 for sure

**Comment by yeeterlol** [comment]
techinically works but it crashes

**Pull request closed without merging**